{
  "file_data": [
    {
      "desc": {
        "cause": "Under Investigation",
        "cluster": false,
        "cust_a": {
          "val": 8
        },
        "etr": "2020-12-18T20:00:00-0400",
        "start": "2020-12-18T10:01:10-0400"
      },
      "geom": {
        "p": [
          "smpoGnnwaK"
        ]
      },
      "id": "1",
      "title": "Outage Information"
    }
  ],
  "file_title": "030322"
}
